src/services/teamService.ts

```diff
@@ -109,7 +109,7 @@ const duplicateName = async (participateTeamDto: participateTeamDto) => {
         name: participateTeamDto.nickname,
       },
     });
-    if (data) {
+    if (!data) {
```

Comment:
```suggestion
    if (data) {
```
데이터가 있을 경우, 중복이 되어야겠지요?!